web/src/app/add-stuff/add-stuff.component.ts

```diff
@@ -27,20 +28,21 @@ export class AddStuffComponent implements OnInit {
 
   stuffFormInit() {
     this.addStuff = this.fb.group({
-      name: '',
+      name: ['', [Validators.required, Validators.pattern('.*\\S.*[a-zA-z0-9 ]')]],
```

Comment:
It's kind of strange regexp. Why do we need it? I think we can avoid this validation. :) 
Also now it gives us "more than 2 characters" and I see the error after the firts letter I entered. I suggest not do it more complex than it should be and just check if the field is not empty without regex.